vich_uploader.xsd

```diff
@@ -0,0 +1,32 @@
+<xs:schema
+        attributeFormDefault="unqualified"
+        elementFormDefault="qualified" xmlns:xs="http://www.w3.org/2001/XMLSchema"
+        targetNamespace="https://vich-uploader-bundle/schema/"
+>
+    <xs:element name="vich_uploader">
+        <xs:complexType>
+
+            <xs:sequence>
+
+                <xs:element name="field">
+                    <xs:complexType>
+                        <xs:simpleContent>
+                            <xs:extension base="xs:string">
+                                <xs:attribute type="xs:string" name="name" use="required"/>
+                                <xs:attribute type="xs:string" name="mapping" use="required"/>
+                                <xs:attribute type="xs:string" name="filename_property" use="required"/>
```

Comment:
I don't use XML mapping very much, but if I remember correctly the `filename_property` should be optional, it should work without it when you have a simple mapping with only a single property.
Could you check it and, in case, remove the `required` property?

Reply:
I think that thhe `filename_property` is required. On the docs it's stated on all mappings as required  (and makes sense it's required, IMO).

It's mentioned [here](https://github.com/dustin10/VichUploaderBundle/blob/master/docs/usage.md#step-2-link-the-upload-mapping-to-an-entity) for attribute mapping, [here](https://github.com/dustin10/VichUploaderBundle/blob/master/docs/mapping/xml.md) for XML mapping, and [here](https://github.com/dustin10/VichUploaderBundle/blob/master/docs/mapping/yaml.md) for YAML mapping.

Reply:
It could be a case of outdated docs. At one point, we introduced that possibility and maybe we forgot to check for all documentation.
Anyway, can you try and tell me if it works without it?

Reply:
Finally got around to try it
.
IMO, it's a rather weird situation.

The `filename_property` config is _not_ mandatory, **but** it will only work if the entity has a an accessible property called exactly `file_namep`.

E.g. if you do not use that configuration, but your entity does not have this property or uses any slightly different name, the upload will fail:

<img width="1088" alt="imagen" src="https://user-images.githubusercontent.com/1815039/205236800-78a643f9-958c-494b-b589-3461113967b4.png">


Since having the property **is** required, I think it makes sense to leave the requirement to configure the mapping correctly, as stated on the existing docs.

Less errors this way.

Reply:
Making it non-required would make necessary to make many more changes on the docs, and explain that this will work if you have an specific name for the this property, which would open the door for more user errors.